### ISSUE TITLE
[Design] TDM Product Images

### ISSUE THREAD

**Issue by Matt-Hood** [Matt-Hood/nes#13]
We will need to make sure all current images on the TDM site are photoshopped to look the best they can possibly look. We really want to make the images on the site "pop" so adding design touches to our assets to get us there will be required. If you think animated gifs would add to the look and feel of the website, feel free to add them.

**Acceptance Criteria** 
All current images on the TDM site are photoshopped and tweaked to optimize their look and feel. Photos will be provided.

**Comment by ExDonald** [comment]
how many photos are we talking about? i could take them all and make them look more appealing and nice. also I don't think that making them as a gif would change anything unless you have something specific you look for?

**Comment by Matt-Hood** [comment]
Around 15 images, plus we don't really have any good images for our bundles. The bundles include multiple meals so we would need you to create photos to represent them. I'll work on getting all the images together. Will be assigning it over tmr, want to make sure we get all the images we need optimized over to you all at once.

**Comment by ExDonald** [comment]
i mean, you are able to take the photos of the meals yourself right? since you actually cook them irl

**Comment by Matt-Hood** [comment]
Yeah we have photos of all the meals, the image for the bundle looks pretty bad though since we didn't have a good way to show that the bundle includes multiple meals in a photo

**Comment by Matt-Hood** [comment]
example: https://tastydiabeticmeals.com/product/15

**Comment by ExDonald** [comment]
okay so it would be required to enhance all the photos of the meals which is around 15 photos and then make a few extra bundle photos which includes a few of the solo meals all in one, correct?

**Comment by Matt-Hood** [comment]
Yeah exactly, you are on point with the requirements. In the future as we introduce new products we will be running the photos by you for enhancements as well.

**Comment by ExDonald** [comment]
so the current photos do not need any enhancements or do they? i know the bundles still need to be made

**Comment by Matt-Hood** [comment]
All photos need enhancements, we really want to make everything look as good as possible

**Comment by Matt-Hood** [comment]
@ExDonald The images have been sent to you on slack

**Comment by Victor1aCP** [comment]
Also I believe some more colour contrasts need to be done in the meals. There is the tasty cauliflower that needs  some colour of red and orange and greens and the tasty chicken with bean sprout salad . I agree with making the meals and taking more images of the meals and if necessary then they can be enhanced .
